app/build.gradle

```diff
@@ -1,10 +1,13 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
     id 'kotlin-kapt'
     id 'dagger.hilt.android.plugin'
     id 'androidx.navigation.safeargs.kotlin'
     id 'com.google.android.gms.oss-licenses-plugin'
+    id 'com.google.devtools.ksp'
```

Comment:
Why it is added during the dependencies bump :D?

Reply:
Because of it 😄
![image](https://github.com/Nataniel-Antosik/Bench_Project/assets/57491794/a1201aef-14fd-4c5e-a4e5-ca29210955f2)

Reply:
If you think it is a good idea I can split this PR to add KSP to the project and bump dependency

Reply:
Yes, it would be nice to split it :)

Reply:
Done 😄